src/Tizen.Applications.Common/Tizen.Applications/CoreApplication.cs

```diff
@@ -191,9 +191,10 @@ protected virtual void OnCreate()
 
             if (!GlobalizationMode.Invariant)
             {
-                string locale = ULocale.GetDefaultLocale();
-                ChangeCurrentUICultureInfo(locale);
-                ChangeCurrentCultureInfo(locale);
+                CultureInfo locale = LocaleManager.GetSystemLocale();
```

Comment:
How about using environment variable for performance?

Reply:
I changed it to use an environment variable. Thank you :)